feature_layers/create-mobile-geodatabase/src/main/java/com/esri/samples/create_mobile_geodatabase/CreateMobileGeodatabaseController.java

```diff
@@ -0,0 +1,326 @@
+      mapView.getGraphicsOverlays().add(graphicsOverlay);
+
+      // create a graphic to add the simple marker symbol in the graphics overlay
+      var simpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.CIRCLE, 0xFF000000, 10);
+      var graphic = new Graphic();
+      graphic.setSymbol(simpleMarkerSymbol);
+      graphicsOverlay.getGraphics().add(graphic);
+
+      // keep track of the points added by the user
+      List<Point> inputs = new ArrayList<>();
+
+      // create a point from user input and add it to the geodatabase feature table
+      mapView.setOnMouseClicked(e -> {
+        if(createGeodatabaseButton.isDisabled() && !isTableWindowOpen) {
+          if (e.isStillSincePress() && e.getButton() == MouseButton.PRIMARY) {
+            // create 2D point from pointer location
+            var point2D = new Point2D(e.getX(), e.getY());
+
+            // create a map point from 2D point
+            Point mapPoint = mapView.screenToLocation(point2D);
+
+            // the map point should be normalized to the central meridian when wrapping around a map, so its value stays within the coordinate system of the map view
+            Point normalizedMapPoint = (Point) GeometryEngine.normalizeCentralMeridian(mapPoint);
+
+            // add a point where the user clicks on the map and update the inputs graphic geometry
+            inputs.add(normalizedMapPoint);
+            Multipoint inputsGeometry = new Multipoint(new PointCollection(inputs));
+            graphic.setGeometry(inputsGeometry);
+
+            if (inputs.size() > 0) {
+              // set up the feature attributes
+              Map<String, Object> featureAttributes = new HashMap<>(Map.of("collection_timestamp", Calendar.getInstance().getTime().toString()));
+              // create a new feature at the map point
+              var feature = geodatabaseFeatureTable.createFeature(featureAttributes, normalizedMapPoint);
+              // add the feature to the feature table
+              var addFeatureFuture = geodatabaseFeatureTable.addFeatureAsync(feature);
+              addFeatureFuture.addDoneListener(() -> {
+                try {
+                  addFeatureFuture.get();
+                  // update the total feature count on screen if feature was added successfully
+                  label.setText("Number of features added: " + geodatabaseFeatureTable.getTotalFeatureCount());
+
+                } catch (InterruptedException | ExecutionException ex) {
+                  throw new RuntimeException(ex);
+                }
+              });
+            }
+          }
+        }
+        else {
+          // close table before adding a new feature to the map
+          handleTableWindowVisibility();
+        }
+      });
+
+    } catch (Exception e) {
+      // on any error, display the stack trace.
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * create geodatabase and feature layer from geodatabase feature table descriptions
```

Comment:
Javadoc (e.g. class or method descriptions) has slightly different styling than inline comments, Javadoc styling is always written as full sentence style. Comment applies for other Javadoc here too

```suggestion
   * Create geodatabase and feature layer from geodatabase feature table descriptions.
   * ```